index.d.ts

```diff
@@ -0,0 +1,61 @@
+export type Replacer = (key: string, value: any) => void;
+export type SortKeys = (a: string, b: string) => number;
+
+export interface Options {
+	/**
+	 * Indentation as a string or number of spaces. Pass in null for no formatting.
+	 *
+	 * @default '\t'
+	 */
+	indent?: string | number | null;
+	/**
+	 * Detect indentation automatically if the file exists.
+	 *
+	 * @default false
+	 */
+	detectIndent?: boolean;
+	/**
+	 * Sort the keys recursively. Optionally pass in a compare function.
+	 *
+	 * @default false
+	 */
+	sortKeys?: SortKeys | boolean;
```

Comment:
I think `boolean` should be first as it should be sorted by most common usage, and most users will only ever need to set `{sortKeys: true}`.

Reply:
👍 sounds good to me